### ISSUE TITLE
near_far, obj_scale, bound 

### ISSUE THREAD

**Issue by Lxiangyue** [facebookresearch/banmo#34]
Hi, thanks again for your awesome work!

Could you please tell me the means and relationships of parameter 'near_far', 'obj_bound', 'obj_scale', 'bound', and 'bound_factor'? I am so confused about these parameters. 

Also, I can understand you set the object center localized at z=0.3 in the world space, and design the 'warmp_shape' to initialize the object as a small sphere by training the SDF. But why set the near_far (initialized by 0-0.6) as a learning parameter (reset_nf) instead of a fixed hyperparameter?

**Comment by gengshan-y** [comment]
Hi, thanks for the questions. 

- near-far (N,2): near and far plane for each image.
- obj_bound (3): a loose bound of the object surface along xyz axis in the canonical space. see B.3 of the [paper](https://banmo-www.github.io/banmo-cvpr.pdf) for details.
- obj_scale: This is a fixed constant and should always equals to 10. Please ignore it.
- bound_factor: The looseness of the initial near_far plane. To cover more spatial points than the initial unit sphere during rendering, we set the near-far planes with a bigger sphere as `bound_factor` * initial unit sphere. We found bound_factor=2 to be a good value and fixed it through experiments.

The reason `near_far` are nn.parameters is to facilitate model loading (there may be a better design). Indeed, `near_far` are not optimized through gradients, but only updated using rest shape and cameras [here](https://github.com/facebookresearch/banmo/blob/1d022945be987a601a36a25c27e71a8c06008bef/nnutils/banmo.py#L453).
